spec/features/cart_show_page_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe 'as user', type: :feature do
+  before(:each) do
+    merchant = User.create!(username: 'test', street: '123 main st', city: 'denver', state: 'CO', zip_code: 80216, email: 'test@bob.net', password: 'password', role: 1)
+    @item = Item.create(name: 'pot', description:'small pot for plants', quantity: 30, price: 2.49, thumbnail: 'https://images-na.ssl-images-amazon.com/images/I/71VGZezvsGL._SX466_.jpg', user_id: merchant.id)
+    @item_2 = Item.create(name: 'crayon', description:'small crayon for plants', quantity: 40, price: 13.5, thumbnail: 'https://images-na.ssl-images-amazon.com/images/I/71VGZezvsGL._SX466_.jpg', user_id: merchant.id)
+    visit item_path(@item)
+    click_button 'Add to Cart'
+    visit item_path(@item)
+    click_button 'Add to Cart'
+    visit item_path(@item_2)
+    click_button 'Add to Cart'
+  end
+  after(:each) do
+
+  end
+
+  it 'should show me all items in my cart' do
+    visit carts_path
+
+    within "#item-#{@item.id}" do
+      expect(page).to have_content("#{@item.name}")
+      #expect(page).to have_xpath("//img[@src='#{@item.thumbnail}]")
```

Comment:
if you throw a CSS id on your images, you can do something like this:

`expect(page.find("#item-#{item.id}-image")['src']).to have_content(item.thumbnail)`